client/client.go

```diff
@@ -80,6 +80,11 @@ func NewClient(c *Config) (*Client, error) {
 	if err != nil {
 		return nil, err
 	}
+	if !(regexp.MustCompile(`slb.com[:\d+]*$`).MatchString(u.Host) ||
+		regexp.MustCompile(`localhost[:\d+]*$`).MatchString(u.Host) ||
+		regexp.MustCompile(`127\.0\.0\.1[:\d+]*$`).MatchString(u.Host)) {
```

Comment:
This condition can be simplified to a more straightforward evaulation, and the `+` isn't needed in the regexp:

```go
regexp.MustCompile(`slb.com|localhost|127\.0\.0\.1)[:\d]*$`)
```

Reply:
Collapsed to `(slb\.com|localhost|127\.0\.0\.1)[:\d]*$`